### ISSUE TITLE
How to define an Array of blackboxed objects?

### ISSUE THREAD

**Issue by derwaldgeist** [aldeed/meteor-simple-schema#476]
I would like to define a property like this:

```js
  participants: {
    type: [Object],
    optional: true
  }
```
It should be an array of arbitrary, blackboxed objects - that means the SimpleSchema should not care about the contents of the objects, and the array as such should be optional, too.

If I set-up the collection schema like this and try to add an array of objects, I always get the following error message:

```
Error: 0 must be an object
```

I also tried to add ```blackbox:true```, but this did not have any effect.

How can I define an array of arbitrary, blackboxed plain objects?

**Comment by derwaldgeist** [comment]
I found out what the problem was: The object I had included in the array was not a plain object, but a custom type. It seems as if this is not an accepted value for [Object]. After cloning the object to a plain object, it worked.

However, I've also noticed a quite strange behaviour: If I do not include ```blackbox:true```, the object handed over to the collection will completely cleared. However, since this object is typically a reference type, it means that this causes unwanted behaviour - since the object handed over to the Collection.insert() method is cleared as well. I would not have expected this. I only found out because it seemed as if my objects were "consumed" by the find() method.
